installer/persistence.go

```diff
@@ -76,3 +76,28 @@ func loadSSHKey(name string) (*sshkeygen.SSHKey, error) {
 	}
 	return key, nil
 }
+
+func loadSSHKeys() ([]*sshkeygen.SSHKey, error) {
```

Comment:
This isn't actually used and can be removed for now. It's use will be for matching any local key in the installer keys dir with one saved on the install target.